src/lib/components/inbox/bids/active_bid.tsx

```diff
@@ -0,0 +1,170 @@
+import { weekdays } from 'moment';
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components/native"
+
+import { View, Text, TouchableWithoutFeedback } from "react-native"
+import { BodyText, Subtitle, MetadataText } from "../typography"
+
+import colors from "../../../../data/colors"
+import OpaqueImageView from "../../opaque_image_view"
+
+const Container = styled.View`
+  margin: 17px 20px 0px;
+`
+
+const Content = styled.View`
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+`
+
+const ImageView = styled(OpaqueImageView) `
+  width: 58px;
+  height: 58px;
+  border-radius: 4;
+`
+
+const MetadataContainer = styled.View`
+  margin: 0 15px;
+  flex: 1;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const Separator = styled.View`
+  height: 1;
+  width: 100%;
+  background-color: ${colors["gray-regular"]};
+  margin-top: 17px;
+`
+
+const StatusLabel = styled(MetadataText) `
+  color: ${props => {
+    switch (props.status) {
+      case "winning": return "rgb(14, 218, 131)"
```

Comment:
are these not the default Artsy red, green, and yellows? That are in the `colors` file? 